pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -35,11 +35,12 @@ import (
 )
 
 const (
-	agentName     = "zedrouter"
-	runDirname    = "/var/run/zedrouter"
-	tmpDirname    = "/var/tmp/zededa"
-	DataPlaneName = "lisp-ztr"
-	DropMarkValue = 0xFFFFFF
+	agentName      = "zedrouter"
+	runDirname     = "/var/run/zedrouter"
+	tmpDirname     = "/var/tmp/zededa"
+	DataPlaneName  = "lisp-ztr"
+	DropMarkValue  = 0xFFFFFF
+	NhProbeInteval = 15
```

Comment:
Please add a story on making this and other constants/timers setable using configItem

Reply:
right. will do.